pkg/crc/version/version.go

```diff
@@ -87,7 +87,12 @@ func GetCRCLatestVersionFromMirror() (*CrcReleaseInfo, error) {
 	client := &http.Client{
 		Timeout: 5 * time.Second,
 	}
-	response, err := client.Get(releaseInfoLink)
+	req, err := http.NewRequest(http.MethodGet, releaseInfoLink, nil)
```

Comment:
Don't we need to use the `CRC_DISABLE_UPDATE_CHECK` env variable to stop this check?

Reply:
The variable is automatically bound to the config key via that config package. It is a magic feature of viper.